app/src/config/default.py

```diff
@@ -0,0 +1,26 @@
+#
+# Default configuration.
+#
+# This is the base layer of configuration. It is used if an environment does not override a value.
+# Each environment may override individual values (see local.py, dev.py, prod.py, etc.).
+#
+# This configuration is also used when running tests (individual test cases may have code to use
+# different configuration).
+#
+
+from src.adapters.db.clients.postgres_config import PostgresDBConfig
+from src.app_config import AppConfig
+from src.logging.config import LoggingConfig
+from src.config import RootConfig
+
+
+def default_config():
+    return RootConfig(
+        app=AppConfig(),
+        database=PostgresDBConfig(),
+        logging=LoggingConfig(
+            format="json",
+            level="INFO",
+            enable_audit=True,
+        ),
+    )
```

Comment:
Would this require any process that runs to be able to load all environment variables? For example, if I have a config for connecting to a separate service, and it's only used in a few select processes, would I put that here in the default config? Let's assume that I don't want to set defaults in the config and want it to error if any fields aren't set.